torch/_subclasses/fake_tensor.py

```diff
@@ -720,6 +725,8 @@ def __init__(
 
         self.shape_env = shape_env
 
+        self.fakified_real_tensors = set()
```

Comment:
this is unused, plz delete. 
```
with FakeTensorMode(allow_non_fake_inputs=True):
     y = x.add_(3)
     print(isinstance(y, FakeTensor) # False
```

The reason we don't need it is because aten codegen returns the input instead of the output here.